numba/cuda/cudadrv/nvvm.py

```diff
@@ -798,7 +903,12 @@ def set_cuda_kernel(lfunc):
 
     # set nvvm ir version
     i32 = ir.IntType(32)
-    md_ver = m.add_metadata([i32(1), i32(2), i32(2), i32(0)])
+    if NVVM().is_nvvm70:
+        ir_versions = [i32(1), i32(6), i32(3), i32(0)]
```

Comment:
Note: this is NVVM IR 1.6 and Dwarf? 3.0

Reply:
Will add a comment, it would certainly help understandability.

Reply:
thanks!